components/MonogramAvatar/MonogramAvatar.jsx

```diff
@@ -8,11 +8,14 @@ import styles from './monogram_avatar.scss'
 
 import Avatar from '../Avatar/Avatar'
 
+type AvatarSize = "smaller" | "small" | "base" | "large" | "larger"
```

Comment:
Here's an example of my limited grasp of modern JS. 

Obviously, for DRY's sake this code should re-use the styles from `Avatar` instead of repeating them here...

Reply:
You can export this type on this file with
`export type AvatarSize = "smaller" | "small" | "base" | "large" | "larger"`

and then import it on the `Avatar` file with 
`import type { AvatarSize } from '../Avatar/Avatar'`

Reply:
Although the prop type is defined in `Avatar`, an specific enum for Avatar size isn't. A good DRY here would be to:
1) move this declaration to `Avatar` (exported: i.e.: `export type AvatarSize = …`)
2) use `AvatarSize` here https://github.com/powerhome/nitro-storybook/blob/master/components/Avatar/Avatar.jsx#L14
3) import `AvatarSize` here with `import type { AvatarSize } from '../Avatar/Avatar'